### DISCUSSION TITLE
ETHUSD is not supported on BITMEX

### DISCUSSION THREAD

**Discussion by Jackjhonson** [bmoscon/cryptofeed#883] [Q&A]
When I load the code, as long as it shows : Loading exchange Bitmex, it will stop and remind me of "cryptofeed.exceptions.
UnsupportedSymbol: ETHUSD is not supported on BITMEX", is there something wrong with the version of cryptofeed.

**Comment by bmoscon** [comment]
that symbols doesnt exist. Here are the current eth- symbols:

'ETH-USD-PERP',
 'ETH-USDT-PERP',
 'ETH-USD-22U30',
 'ETH-BTC-22U30',
 'ETH-USDT-22U30',
 'ETH-BTC-22Z30',
 'ETH-USD-22Z30',
 'ETH-USDT-22Z30',
